client/src/Components/GoogleMap/Google_Secret.js

```diff
@@ -0,0 +1,2 @@
+const GOOGLE_MAPS_API_KEY = 'AIzaSyB05HDuP6Jvdsy8jUcnvvkrW-Qq0Uluoyw';
```

Comment:
can we add a comment here: ```// NB Google Maps API key is not secret. It's fine to have it in clear text
// because we whitelist URLs from the google maps console
// If someone tries to use our key, they won't be able to use it anywhere except localhost
```

Also the file should be changed to `client/src/config.js`. It's not a component.